scripts/README.md

```diff
@@ -1,5 +1,5 @@
 tsplot
```

Comment:
because this is markdown, how about making these headers, i.e. 
```
tsplots
=====
```
can be replaced with
```
#tsplots
```